core/command/__tests__/command.test.js

```diff
@@ -387,5 +387,24 @@ describe("core-command", () => {
         })
       );
     });
+
+    it("throws ENOWORKSPACES when npm client is pnpm and useWorkspaces is not true", async () => {
+      const cwd = await initFixture("pnpm");
+
+      const lernaConfigPath = path.join(cwd, "lerna.json");
+      const lernaConfig = await fs.readJson(lernaConfigPath);
+      await fs.writeJson(lernaConfigPath, {
+        ...lernaConfig,
+        useWorkspaces: false,
+      });
+
+      await expect(testFactory({ cwd })).rejects.toThrow(
+        expect.objectContaining({
+          prefix: "ENOWORKSPACES",
+          message:
+            "Usage of pnpm without workspaces is not supported. To use pnpm with lerna, set useWorkspaces to true in lerna.json and configure pnpm to use workspaces: https://pnpm.io/workspaces.",
```

Comment:
Same comment regarding trailing period and overall consistency of error formatting for these